package.json

```diff
@@ -108,10 +101,9 @@
     "react-test-renderer": "^16.4.2",
     "shx": "^0.3.2",
     "stylelint": "^9.2.0",
-    "stylelint-config-terra": "^1.0.0",
+    "stylelint-config-terra": "^2.0.0",
     "terra-base": "^4.0.0",
-    "terra-dev-site": "^2.0.0",
-    "terra-toolkit": "^4.16.0",
-    "xfc": "^1.2.1"
```

Comment:
I believe we need to keep xfc for the terra-embedded-content-consumer package:
https://github.com/cerner/terra-framework/blob/master/packages/terra-embedded-content-consumer/package.json#L27

Reply:
Ah, we defined xfc in terra-dev-site, so I didn't realize we needed to defining it here. Makes sense. 👌 will re-add